tests/cli.test.js

```diff
@@ -0,0 +1,10 @@
+jest.mock('fs')
+
+beforeEach(() => {
+  require('fs').__reset()
+})
+
+test.todo('prints help when no arguments are passed')
+test.todo('prints help when --help is passed')
+test.todo('fails to start creating project in a directory that already exists')
+test.todo('creates project with labels passed to Dockerfile from questionnaire')
```

Comment:
The idea for this test implementation is for it to look roughly something like this:

```js
await runCLI(['action-directory-name'])
expect(mockFs.__getContent('action-directory-name/Dockerfile')).toMatchSnapshot()
expect(mockFs.__getContent('action-directory-name/package.json')).toMatchSnapshot()
expect(mockFs.__getContent('action-directory-name/entrypoint.js')).toMatchSnapshot()
```

With the filesystem mock in `__mocks__/fs.js`, we can store file path and content data in memory, access it in a test, and execute whatever assertions we want on it.

Thoughts on this approach?

Reply:
I haven't used Jest's module mocking much - typically when I need to use `fs` methods in tests, I just have it read/write actual files (maybe to a `tmp` folder or something). I think this is probably a better method though 🤔 Works for me!

Side-note: never knew about `test.todo`! That's awesome 😁